.rubocop.yml

```diff
@@ -1,11 +1,16 @@
 Rails:
- Enabled: true
+  Enabled: true
 AllCops:
- TargetRubyVersion: 2.2.3
+  Exclude:
```

Comment:
@sashank1995 Add `Rakefile`, `config/environments/*`,`config/initializers/devise.rb`, `config/application.rb`, `config/initializers/wrap_parameters.rb`, `spec/spec_helper.rb`, and `spec/rails_helper.rb` under exclude.